app.py

```diff
@@ -1,26 +1,197 @@
-from modules.Preprocessing import get_tweet_data
-from modules.Sentiment_Analysis import train_model_if_necessary, analyze_many_tweets
-
-# Train model
-train_model_if_necessary()
-# scrape here
-print('Scraping...')
-tweet_data = [{}]  # do stuff
-# open file for preprocessing
-print('Preprocessing...')
-# FILENAME = './'
-# with open(FILENAME) as file_object:
-#     tweet_data = file_object.read()
-
-handled_tweets = get_tweet_data(tweet_data)  # do stuff
-
-# analysis here
-print('Performing sentiment analysis...')
-analyzed_tweets = *handled_tweets  # do stuff
-
-# presentation here
-print('Generating presentations...')
-result = *analyzed_tweets  # do stuff
+import datetime
+from modules.web_scraper import get_tweets
+from modules.Preprocessing import handle_tweet_data
+# from modules.Preprocessing import handle_tweet_data
+# from modules.Sentiment_Analysis import train_model_if_necessary, analyze_many_tweets
+# import modules.presentation as presentation
+import argparse
+
+
+#########CUSTOM TYPES#########
+def _restricted_float(val: float):
+    try:
+        val = float(val)
+    except ValueError:
+        raise argparse.ArgumentTypeError(
+            f"{val} not a floating-point literal")
+
+    if 0.0 < x > 1.0:
+        raise argparse.ArgumentTypeError(f"{val} not in range [0.0, 1.0]")
+    return val
+
+
+#########CUSTOM TYPES#########
+
+#########HELPER METHODS#########
+
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawTextHelpFormatter):
+    """
+    Allows us to use two formatters for argparse instead of only the default 1 (hackish)
+    https://stackoverflow.com/a/18462760
+    """
+
+    def _split_lines(self, text, width):
+        # https://stackoverflow.com/a/29498894
+        # modify super to add newline between arguments
+        lines = super()._split_lines(text, width)
+        if text.endswith('\n'):  # custom newline
+            lines += ['']
+        else:
+            lines += ['']  # also add newline between different arguments
+        return lines
+
+
+def _default_date():
+    """
+    Generates today and five days from now for use with default values of the date-argument
+    """
+    today = datetime.date.today()
+    five_days_from_now = today + datetime.timedelta(days=5)
+    # create readable format, as should be input
+    return ' '.join([today.strftime('%Y-%m-%d'), five_days_from_now.strftime('%Y-%m-%d')]) + ' (next 5 days)'
```

Comment:
This `(next 5 days)` should be changed, as it's most likely going to interfere with date-parsing at a later point in the program.